beta/test_run_samples.py

```diff
@@ -0,0 +1,64 @@
+#  Copy this Python script to the root dir and run it to:
+#     - load each sample project (from user_projects)
+#     - compile it
+#     - modify each project's .xml:  max_time, # threads, output folder
+#     - run it
+#
+#  WARNING: this is primarily intended to be used by core developers when testing a new release.
+#            It will create new output directories  ("output_<proj>")
+#
+#  Reminder:
+# $ ls user_projects/
+# biorobots/		hetero/			pred_prey/		worm/
+# cancer_biorobots/	interaction/		rules/
+# cancer_immune/		mechano/		template/
+# celltypes3/		physimess/		virus_mac/
+
+# Sample projects: template biorobots-sample cancer-biorobots-sample cancer-immune-sample
+#                  celltypes3-sample heterogeneity-sample pred-prey-farmer virus-macrophage-sample
+#                  worm-sample interaction-sample mechano-sample rules-sample physimess-sample
+#
+#  It does not currently test running the intracellular projects:
+# Sample intracellular projects: template_BM ode-energy-sample physiboss-cell-lines-sample
+#                  cancer-metabolism-sample physiboss-tutorial physiboss-tutorial-invasion
+#
+#
+#   If you want to cleanup the created execs:
+# $ rm -rf template biorobots cancer_biorobots cancer_immune celltypes3 hetero pred_prey virus_mac worm interaction mechano rules physimess
+
+import subprocess
+import xml.etree.ElementTree as ET
+import os
+import time
+
+# skip over the 3D cancer_immune_sample for now
```

Comment:
add context so we know when to include that project in testing?

Reply:
since you said you were going to add comments to it, I would also appreciate clarity on this one :)